Examples/Python/src/Detray.cpp

```diff
@@ -13,13 +13,12 @@
 #include <memory>
 #include <string>
 
+#include <detray/builders/detector_builder.hpp>
+#include <detray/io/frontend/detector_reader_config.hpp>
```

Comment:
Not sure we need the file reader config for anything?
```suggestion
```

Reply:
I changed it.